melodies_monet/driver.py

```diff
@@ -1186,6 +1201,11 @@ def plotting(self):
         -------
         None
         """
+        
+        from .util.tools import resample_stratify
+        from .util.tools import vert_interp
+        import matplotlib.dates as mdates
+        import numpy as np
```

Comment:
Do you use all of these? I looks like numpy is already pulled in at the top. mdates is not used and vert_interp is not used.